src/components/DiveLog.js

```diff
@@ -1,44 +1,97 @@
 import styled from 'styled-components';
+import { useState } from 'react';
 
 export default function DiveLog({
   divenumber,
   city,
   country,
   locationname,
   date,
+  typeOfDive,
+  timeIn,
+  timeOut,
+  bottomTime,
+  maxDepth,
+  buddyName,
+  notes,
 }) {
+  const [active, setActive] = useState(true);
   return (
-    <>
-      <DiveLogCard>
-        <DiveNumber>2</DiveNumber>
-        <span>
-          <p>Miami, USA</p>
-          <p>Offshore</p>
-        </span>
-        <p>25.06.2015</p>
-      </DiveLogCard>
-      <DiveLogCard>
-        <DiveNumber>{divenumber}</DiveNumber>
-        <span>
-          <p>
-            {city}, {country}
-          </p>
-          <p>{locationname}</p>
-        </span>
-        <p>{date}</p>
-      </DiveLogCard>
-    </>
+    <li>
+      {active && (
+        <DiveLogCard id="divelog" onClick={() => handleCardToggle()}>
+          <DiveNumber>{divenumber}</DiveNumber>
+          <div>
+            <p>
+              {city}, {country}
+            </p>
+            <p>{locationname}</p>
+          </div>
+          <div>
+            <p>{date}</p>
+            <p>{typeOfDive}</p>
+          </div>
+        </DiveLogCard>
+      )}
+
+      {!active && (
+        <DiveLogDetailsCard onClick={handleCardToggle}>
+          <DiveNumberDetails>{divenumber}</DiveNumberDetails>
+          <DetailsCardHead>
+            <div>
+              <p>
+                {city}, {country}
+              </p>
+              <p>{locationname}</p>
+            </div>
+            <div>
+              <p>{date}</p>
+              <p>{typeOfDive}</p>
+            </div>
+          </DetailsCardHead>
+          <CardDetails>
+            <p>time in: </p>
+            <p>{timeIn}</p>
+            <p>time out: </p>
+            <p>{timeOut}</p>
+            <p>bottom time: </p>
+            <p>{bottomTime}</p>
+            <p>max. depth: </p>
+            <p>{maxDepth}</p>
+            <p>buddy: </p>
+            <p>{buddyName}</p>
+            <p>notes: </p>
+            <p>{notes}</p>
+          </CardDetails>
+        </DiveLogDetailsCard>
```

Comment:
I think you can shorten your code here, so that you always render the DetailsCardHead( if it is the same as i guess) and only conditionally render the CardDetails  🐜